.github/workflows/cpp.yml

```diff
@@ -24,7 +24,7 @@ jobs:
           wget https://apache.jfrog.io/artifactory/arrow/$(lsb_release --id --short | tr 'A-Z' 'a-z')/apache-arrow-apt-source-latest-$(lsb_release --codename --short).deb
           sudo apt install -y -V ./apache-arrow-apt-source-latest-$(lsb_release --codename --short).deb
           sudo apt update
-          sudo apt install -y -V libarrow-dev libarrow-dataset-dev libparquet-dev
+          sudo apt install -y -V libarrow-dev=9.0.0-1 libarrow-dataset-dev=9.0.0-1 libparquet-dev=9.0.0-1
```

Comment:
let's just make this one variable? we're going to have to do this again in a few months lol

Reply:
sure will do.